stack_configuration_summary_integration_test.go

```diff
@@ -27,7 +27,7 @@ func TestStackConfigurationSummaryList(t *testing.T) {
 	stack, err := client.Stacks.Create(ctx, StackCreateOptions{
 		Name: "aa-test-stack",
 		VCSRepo: &StackVCSRepoOptions{
-			Identifier:   "hashicorp-guides/pet-nulls-stack",
+			Identifier:   "ctrombley/linked-stacks-demo-network",
```

Comment:
```suggestion
			Identifier:   "hashicorp-guides/pet-nulls-stack",
```

Caught some vcs repo identifiers changed to your personal ones (your own testing I'd imagine). We should get these reverted back.